### PR TITLE
Fix alphabetical order of properties

### PR DESCRIPTION
**Pull request by ntacheva**

### CODE REVIEW

**Comment by svdimitr** [comment]
I never knew we followed alphabetical order when listing the features/attributes/parameters of the component. Let's not merge this as when we revamp the overview articles this would be out of order either way.

**Review by dimodi — APPROVED** [review]
On second thought, we may also experiment with sorting by importance (Value and DebounceDelay first)

**Pull request merged**
